site/src/blog/announcing-typechat-0-1-0.md

```diff
@@ -0,0 +1,206 @@
+
+1. The schema was fixed. While possible to generate a text schema on the fly, it's an error-prone task.
+
+While there are a lot of ergonomic benefits to using a textual TypeScript schema, we explored whether there we could add a bit more flexibility and made a few changes to TypeChat.
+
+The first is that we've broken out a piece of `TypeChatJsonTranslator` into a more granular concept: a `TypeChatJsonValidator`.
+A `TypeChatJsonValidator` is responsible for generating a string schema representation to guide language models, and to actually make sure the data that comes back matches some type.
+This means that to construct a `TypeChatJsonTranslator`, you need to make a `TypeChatJsonValidator` first;
+but it also means that validators are swappable.
+Here's what using that looks like now:
+
+```ts
+import fs from "fs";
+import path from "path";
+
+import { createLanguageModel, createJsonTranslator } from "typechat";
+import { createTypeScriptJsonValidator } from "typechat/ts";
+
+import { SentimentResponse } from "./sentimentSchema";
+
+const model = createLanguageModel(process.env);
+const schema = fs.readFileSync(path.join(__dirname, "sentimentSchema.ts"), "utf8");
+const validator = createTypeScriptJsonValidator<SentimentResponse>(schema, "SentimentResponse");
+const translator = createJsonTranslator(model, validator);
+
+translator.translate("hello world!").then(response => {
+    if (!response.success) {
+        console.log(response.message);
+        return;
+    }
+    console.log(`The sentiment is ${response.data.sentiment}`);
+});
+```
+
+Notice that instead of passing the schema into `createJsonTranslator`, we're passing it into `createTypeScriptJsonValidator` which we need to import from `typechat/ts`.
+The created validator the needs to be passed into `createJsonTranslator`.
+
+For existing calls to `createJsonTranslator`, you'll probably see a message like:
+
+> TS2554: Expected 2 arguments, but got 3.
+
+you'll need to drop the name of the type, and substitute the argument schema with a validator.
+Here's the effective diff:
+
+```diff
+  import { createJsonTranslator, createLanguageModel, processRequests } from "typechat";
++ import { createTypeScriptJsonValidator } from "typechat/ts";
+  import { SentimentResponse } from "./sentimentSchema";
+
+...
+
+- const translator = createJsonTranslator<SentimentResponse>(model, schema, "Sentiment")
++ const validator = createTypeScriptJsonValidator<SentimentResponse>(schema, "SentimentResponse");
++ const translator = createJsonTranslator(model, validator);
+
+...
+```
+
+## Zod Validators
+
+The second change builds on pluggable validators: TypeChat makes it possible to create validators from Zod schemas.
+[If you're not familiar with Zod](https://zod.dev/), it's a fairly popular library in the TypeScript/JavaScript ecosystem for validating data.
+One strength of this library is that as Zod type validator objects are constructed, static types can be derived from them.
```

Comment:
Isn't the implication of zod that you also can now have dynamic schemas you generate at runtime? I think you're getting at that here but it's worth connecting that dot explicitly for readers